File: Service/Musical.Broccoli.API/src/Business/Connectors/UserConnector.cs
using Common.DTOs;
using DataAccessLayer.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using AutoMapper;
using DataAccessLayer.Repositories.Contracts;
using Business.Validators;

namespace Business.Connectors
{
    public class UserConnector : BaseConnector<UserDTO, User>
    {
        public UserConnector(IUserRepository repository, IMapper mapper) : base(repository, mapper)
        {
        }

        public override ValidationResult Validate(UserDTO dto)
        {
            throw new NotImplementedException();
        }
    }
}

File: Musical-Broccoli-App/app/src/main/java/com/cetys/dreamteam/musicalbroccoli/business/connectors/contracts/UserConnector.java
package com.cetys.dreamteam.musicalbroccoli.business.connectors.contracts;

import com.cetys.dreamteam.musicalbroccoli.core.models.User;

/**
 * @Author J. Pichardo on 12/7/2016.
 */

public interface UserConnector extends BaseConnector<User> {

    void login(User user);
}
